feature/sflow/otg_tests/sflow_base_test/README.md

```diff
@@ -0,0 +1,67 @@
+# SFLOW-1: sFlow Configuration and Sampling
+
+## Summary
+
+Verify configuration of sflow and sFlow sample data.
+
+## Procedure
+
+*   SFLOW-1.1 Configure sFlow on DUT
+    * Configure DUT to send sflow samples to ATE port 2
+
+*   SFLOW-1.2 Send traffic via OTG and verify sFlow packet on OTG
+    *   Configure ATE to generate ipv4 and ipv6 traffic and capture sFlow packets
+    *   Use sample rate of 1/1M and sample size of 256 bytes
+    *   Verify captured packet is formatted like an sFlow packet
```

Comment:
- should there be a verification in place to check for the sample size to be 256B?
- should there be a check in place that ensures vendors are complying by the requirement of 1 sample / 1M packets?
- Should we set and check if the device is sending samples w/ DSCP=32?

Whilst the test here seems rudimentary. Actual test would be to check for the following and should be added at a later date.
1. Sampled interface is reported accurately
2. Ingress and Egress interfaces of the flow are accurately reported in the sample
3. If the following contents of the "Extended-router" container are accurate for different flow types, Naked IP v/s IP-IP encaped flows

- Next hop
- Next hop source mask
- Next hop destination mask

Reply:
Good suggestions, I added the verifications.  Since I am writing my own packet decoder (OTG does not provide sflow decoder), I broke out the more advanced decodes into a separate sub-test.